tests/cluster/cluster.go

```diff
@@ -101,7 +101,7 @@ func (api *ClusterAPI) CreatePrivilegedPodWithIP(podName, nadName, bridgeName, c
 		Annotations: map[string]string{"k8s.v1.cni.cncf.io/networks": nadName},
 	},
 		Spec: corev1.PodSpec{Containers: []corev1.Container{{Name: "test",
-			Image:           "alpine",
+			Image:           "quay.io/oshoval/alpine",
```

Comment:
Take official Fedora instead if there is no official Alpine/Cirros repo on Quay.